### PR TITLE
Vet-Clinic-database: Query multiple tables

### PR DESCRIPTION
**Pull request by degisew**
### Vet-Clinic-database: Query multiple tables

**In this PR, I implemented the following project requirements**

[shoot 1](https://github.com/degisew/Vet-clinic-database/blob/query-multiple-tables/shoots/q1.PNG)
[shoot 2](https://github.com/degisew/Vet-clinic-database/blob/query-multiple-tables/shoots/q2.PNG)
[shoot 3](https://github.com/degisew/Vet-clinic-database/blob/query-multiple-tables/shoots/q3.PNG)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,7 +41,9 @@ Give a ⭐ if you like this project!
 
 This project is [MIT](./MIT.md) licensed.
 
-
+![](https://github.com/degisew/Vet-clinic-database/blob/query-multiple-tables/shoots/q1.PNG)
+![](https://github.com/degisew/Vet-clinic-database/blob/query-multiple-tables/shoots/q2.PNG)
+![](https://github.com/degisew/Vet-clinic-database/blob/query-multiple-tables/shoots/q3.PNG)
```
Comment (alexander16108) [inline README.md:46]: hey there, ur project looks good but u need one more step to go further.
the links to your queries are broken so kindly fix them and try again or you can add a screen shot of your queries by using the snipping tool on you pc.

**Review by alexander16108 — CHANGES_REQUESTED** [review]
Hi @degisew , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!





## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (degisew) [reply README.md:46]: Okay, thanks. I will fix them.

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @degisew 

Your project is complete!  🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- screenshots ok

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**